pom.xml

```diff
@@ -63,6 +57,67 @@
 
 	<profiles>
 		<profile>
+			<id>ossrh</id>
+			<distributionManagement>
```

Comment:
If you want to use the basic mvn clean deploy without the release/nexus plugins you will need it, but I have never used the nexus plugin, so it may define all of that internally and you may be fine. :)